pkg/cli/admin/release/info.go

```diff
@@ -1165,10 +1165,98 @@ func describeReleaseDiff(out io.Writer, diff *ReleaseDiff, showCommit bool, outp
 			}
 		})
 	}
+
+	printFeatureSetSection(diff, "Default", w)
+	printFeatureSetSection(diff, string(configv1.TechPreviewNoUpgrade), w)
+
 	fmt.Fprintln(w)
 	return nil
 }
 
+func printFeatureSetSection(diff *ReleaseDiff, featureSetName string, w io.Writer) {
+	newFeatures, newlyEnabledFeatures, newlyDisabledFeatures, newlyRemovedFeatures, _ := featureGatesForFeatureSetChanged(diff, featureSetName)
+	fmt.Fprintln(w)
+	fmt.Fprintf(w, "FeatureGates for %q (%d enabled, %d disabled, %d removed)\n",
+		featureSetName, len(newlyEnabledFeatures), len(newlyDisabledFeatures), len(newlyRemovedFeatures))
+	fmt.Fprintf(w, "Enabled:\n")
```

Comment:
nit: possibly skip this section if `newFeatures` (or `newlyEnabledFeatures`?) is empty?  And same for the other sections.

Reply:
> nit: possibly skip this section if `newFeatures` (or `newlyEnabledFeatures`?) is empty? And same for the other sections.

Agreed.  I'm going to rewrite for diff only.